sample/helpers.py

```diff
@@ -0,0 +1,19 @@
+import re
```

Comment:
You need not create a separate file for `helpers`. Better keep them in the same file as `core.py` outside `WikiPage` class and prefix them with `_`. Names starting with `_` are supposed to refer to functions that should not be messed with by the user unless the user is sure what they are doing. ["We're all consenting adults here"](http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html#naming) :wink: